src/voip_patrol/voip_patrol.cc

```diff
@@ -637,20 +643,22 @@ void TestAccount::onIncomingCall(OnIncomingCallParam &iprm) {
 	prm_100.statusCode = PJSIP_SC_TRYING;
 	call->answer(prm_100);
 
-	LOG(logINFO) <<__FUNCTION__<<"code:" << code <<" reason:"<< reason;
+	LOG(logINFO) <<__FUNCTION__<<": code:" << code <<" reason:"<< reason;
+	if (ring_duration > 0) {
+		prm.statusCode = PJSIP_SC_RINGING;
+		if (early_media) {
+			prm.statusCode = PJSIP_SC_PROGRESS;
+		}
+		call->answer(prm);
```

Comment:
Isn't it a duplication for `call->answer()` that is executed below?

Reply:
this one sets answer with either 180/183.  
one below sets answer with 200. 
without this, 200 was not sent and 180/183 became final response.